align/primitive/main.py

```diff
@@ -257,7 +258,8 @@ def generate_primitive(block_name, primitive, height=28, x_cells=1, y_cells=1, p
     if 'Cap' in primitive:
         blockM = 1
     else:
-        blockM = 0         
+        blockM = 0
+    positive_coord.json_pos(outputdir / (block_name + '.json'))         
```

Comment:
Why are we doing this as a post-processing step? I recommend doing this before writing out the JSON @arvuce22.

Reply:
Actually the assumption that the bounding box always starts at 0,0 may not be valid in the long run (what if we wish to support overlaps between adjoining transistors for example). Its possible this should be handled within the primitive generator natively.

Reply:
@arvuce22 this PR has been merged but would like this to be clarified.